README.md

```diff
@@ -112,8 +112,11 @@ their combination.
 
 We also have explicit _non-goals_ for Carbon, notably including:
 
--   A stable ABI for the entire language and library
+-   A stable [application binary interface](https://en.wikipedia.org/wiki/Application_binary_interface) (ABI) for the entire language and library
 -   Perfect backwards or forwards compatibility
+-   Support legacy code for which source code is unavailable
+-   Follow C++'s compilation and linking model
+-   Handling all C++ code
```

Comment:
I don't quite agree with these three.

The first addition I think *is* a goal, but maybe not the top goal. I think it requires a lot more nuance which the interop strategy gets into and we can't really cover here.

The second one I think is likely to be misunderstood. A goal is to work with existing C++ build systems, and that means we need at least a largely similar compilation and linking model. How much we can diverge there I don't think is clear yet, and so I wouldn't put this as a non-goal. I worry it would be confusing at best. =/

The last one I think is technically true, but likely to be misleading. I feel like the longer goals document talks about this with more of the nuance it needs.

Still, interested in other thoughts about these additions. I think these would at least need to be a proposal as I think it might be a more substantial change that should have a quick rationale and the leads sign off on it. The fix above though should just move forward quickly as a simple wording improvement.

Reply:
-> https://github.com/carbon-language/carbon-lang/pull/1698

Reply:
> The last one I think is technically true, but likely to be misleading. I feel like the longer goals document talks about this with **more of the nuance it needs.**

I will try to formulate this more nuanced into the bullet points. Will read https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2020/p2137r0.html first, to understand better.

Reply:
> The first addition I think is a goal, but maybe not the top goal.

![image](https://user-images.githubusercontent.com/1167114/180833812-1a368143-6f5c-4969-8210-d4347656db44.png)

This point is actually verbatim taken from the first sentence, which is the same as in the `goals.md` document here.

> The second one I think is likely to be misunderstood. 
Agreed.

> The last one I think is technically true, but likely to be misleading.

Alternative suggestion:

```md
-   Stringently support legacy libraries for which source code is unavailable
-   Prioritize compilation and linking model over needs of C++
-   Adhere to or reproduce all C++ semantics
```